CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 ## Unreleased
 
+## 0.7.4
+Released 2019-09-30
+
+- Updated `azure` module
+  ([#773](https://github.com/census-instrumentation/opencensus-python/pull/773))
+  ([#767](https://github.com/census-instrumentation/opencensus-python/pull/767))
```

Comment:
This will render as `((#773) (#776))`, do this instead to match the formatting for other releases with multiple PRs:

```
  ([#773](https://github.com/census-instrumentation/opencensus-python/pull/773),
   [#767](https://github.com/census-instrumentation/opencensus-python/pull/767))
```